sklearn/metrics/_pairwise_distances_reduction/_datasets_pair.pyx.tp

```diff
@@ -55,6 +55,7 @@ cdef class DatasetsPair{{name_suffix}}:
         cls,
         X,
```

Comment:
`X` and `Y` might now be `None` as well.

Reply:
Yes, thank you.